### DISCUSSION TITLE
Dependencies upgraded together

### DISCUSSION THREAD

**Discussion by igorwojda** [renovatebot/renovate#18344] [Request Help] [answered]
I have two dependencies
1. Kotlin (`org.jetbrains.kotlin:kotlin`)
2. Kotlin Symbolic Processing (`com.google.devtools.ksp`)

kotlinSymbolProcessing versioning is linked to Kotlin version eg. for Kotlin `1.7.20` ksp `1.7.20-1.0.7` must be used, so these dependencies have to be upgraded together

(All ksp versions can be found here https://repo.maven.apache.org/maven2/com/google/devtools/ksp/symbol-processing-gradle-plugin/)

Can renovate be configured to update these dependencies together?

**Comment by viceice** [comment, marked as answer]
sure, search the docs or discussions for grouping

**Comment by igorwojda** [comment]
Grouping seems like a way to go, but I can't figure out the exact config. Can you help with that?

**Reply by rarkins** [reply]
Provide your current config/attempt

**Reply by igorwojda** [reply]
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "lockFileMaintenance": {
    "enabled": true,
    "automerge": true,
    "automergeType": "pr",
    "platformAutomerge": true
  },
  "packageRules": [
    {
      "matchUpdateTypes": [
        "minor",
        "patch"
      ],
      "matchCurrentVersion": "!/^0/",
      "automerge": true
    }
  ]
}
```

**Reply by rarkins** [reply]
You want to add an additional package rule with `matchPackageNames` plus `groupName` for whatever dependencies you want to group together. You can see the package names if you look into the `packageFiles with updates` log message
